ElasticFrameProtocol.h

```diff
@@ -396,6 +396,7 @@ class ElasticFrameProtocolReceiver {
         uint8_t mStreamID = 0;           // A streamID used for stream separation of same content type (if you got more than one H264 streams for example)
         uint8_t mSource = 0;             // A transparent value 'passed by' the receivedFragment method to separate multiple parallel EFP streams
         uint8_t mFlags = NO_FLAGS;       // Flags used by the frame
+        uint16_t mSuperFrameNo = 0;      // The 16 bit super frame counter. Useful for calculating the number of lost frames in HOL mode
```

Comment:
Maybe not for this PR, but looking at the repo (this file and the general README.md), there is no real good explanation of HOL as I see it, we don't really explain any "modes" either ... so using it (the abbriviation) here might be fine, but me might consider some other PR where this is explained more in detail in the README for the user?

Reply:
It is explained pretty well in the PDF linked from the README.md (https://raw.githubusercontent.com/agilecontent/efp/master/docs/ElasticFrameProtocol.pdf), but I agree, that we could add some more information about it in the README as well.